terria/africa.json

```diff
@@ -30,6 +30,7 @@
                       "linkedWcsCoverage": "ls5_usgs_sr_scene",
                       "ignoreUnknownTileErrors": true,
                       "opacity": 1,
+					  "featureTimesProperty": "data_available_for_dates",
```

Comment:
indentation fix, and for all below

Reply:
Sorry, new editor on Win10 machine put tabs in